### PR TITLE
RAIL-1997: Added info on settings up SSO

### PR DESCRIPTION
**Pull request by evgeniabogranskaya**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/202/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/66/ : SUCCESS in 6s

File: docs/sso.md [hunk 1]
```diff
@@ -22,15 +22,69 @@ Before you start, verify that your site meets the following requirements:
...
+        }
+        ```
+
+    The following is an example of login code:
+
+        ```javascript
+        import sdk from '@gooddata/gooddata-js';
+        import qs from 'qs';
+
+        const relayState = 'relay-state';
```
Comment (no23reason) [inline docs/sso.md]: Maybe we should use some more descriptive value:

```js
const relayState = 'https://your.application.com';
```

Comment (evgeniabogranskaya) [reply docs/sso.md]: Fixed!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/203/ : SUCCESS in 1m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/67/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/204/ : SUCCESS in 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/62/ : SUCCESS in 14s

**Pull request merged**